src/libraries/System.Drawing.Common/src/System/Drawing/ImageAnimator.Unix.cs

```diff
@@ -131,20 +131,20 @@ public static void StopAnimate(Image image, EventHandler onFrameChangedHandler)
 
             if (ht.ContainsKey(image))
             {
-                AnimateEventArgs evtArgs = (AnimateEventArgs)ht[image];
-                evtArgs.RunThread.Abort();
+                AnimateEventArgs evtArgs = (AnimateEventArgs)ht[image]!;
+                evtArgs.RunThread!.Abort();
                 ht.Remove(image);
             }
         }
 
         public static void UpdateFrames()
         {
-            foreach (Image image in ht.Keys)
-                UpdateImageFrame(image);
+            foreach (Image? image in ht.Keys)
+                UpdateImageFrame(image!);
         }
 
 
-        public static void UpdateFrames(Image image)
+        public static void UpdateFrames(Image? image)
```

Comment:
Above methods `Animate(Image image, EventHandler onFrameChangedHandler),  CanAnimate(Image image), StopAnimate(Image image, EventHandler onFrameChangedHandler)` allowing null `image`, might be not intentional but if we allowing null here i think all others should be same

Reply:
I see you make them nullable in windows, i think for consistency Linux better to be same

Reply:
I'm wondering whether we should be doing the opposite. Linux explicitly states in the comment that arguments must be non-null and both implementations just silently return if the argument is null. I think this strongly indicates that the contract requires non-nullability, with the exception perhaps of `CanAnimate`.

Reply:
Agree, we should go with whichever is more strict